cmd/credential.go

```diff
@@ -29,7 +29,7 @@ const (
 	accessKeyMinLen = 5
 
 	// Maximum length for Minio access key.
-	accessKeyMaxLen = 20
+	accessKeyMaxLenMinio = 20
```

Comment:
For consistency purposes I suffixed it, just like the original code had it for `secretKeyMaxLen**Minio**`.
The other implication by having it there is the fact that max length is needed when generating Minio access and secret keys. So, it is kind of internal. In other words, we don't use them for validating user provided access and secret keys.